src/components/Gallery/components/GalleryHeader/MobileGalleryActions/MobileGalleryActions.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
```

Comment:
let's put all the nested components in the `Gallery/components` directory without unnecessary nesting. Now the nesting is inconsistent, because something is just inside the component directory, and something is inside the additional directory `components`